### DISCUSSION TITLE
Test from vk4tux

### DISCUSSION THREAD

**Discussion by vk4tux** [martinbradford/101Cats#2] [General]
..

**Comment by martinbradford** [comment]
Excellent! Since GitHub is free for low use and only $4 per month for a PRO subscription, I suggest that we transfer here for distribution and discussion.

**Reply by I2BRT** [reply]
![My101cats](https://github.com/martinbradford/101Cats/assets/168969628/ba44de71-351b-4fb1-bbf6-2f004f28413a)
73 de Rick I2BRT

**Reply by jeff-patterson34721** [reply]
Great piece of software!

I've had some issues with some sliders going in to oscillation. Maybe an event storm issue. I can reproduce it by turning on Cont  and adjusting the APF slider.

Suggestion: Swap the position of the NB Level and Contour Freq sliders so that all Contour controls are grouped in one section. 

There's a button labeled "button" next to the TX PTT. It brings up the MODSOURCEDIALOG window.

I'm an experienced C# WPF programmer. If that's the language you're using I'd love to collaborate on this project.

**Reply by martinbradford** [reply]
OK Jeff - I've added you to the collaborators list for the source code repository. I will welcome assistance - take a look at it and ask me questions. I do need to add some documentation - I started this for my own use and did not go out of my way to make it accessible to others!

Regarding your other comments - can you clarify the process for getting a slider to oscillate? That is a bug which I had on several sliders in earlier releases, but I cannot reproduce it here on the current release.

The "button" button is currently under development - the Mod Sources Dialogue box is one of the most recent additions and is still under development and testing.

The **NB Level** and **Contour Frequency** sliders are positioned as they are because **NB Level** is common to both receivers while each receiver has its own **Contour Frequency**.  The entire lower panel of the main window contains controls that are shared between both receivers.

**Comment by W1SWL** [comment]
And here is what a REPLY looks like....... Art

**Comment by jeff-patterson34721** [comment]
Thanks Martin I’ll take a look.

I’ve had a some more trouble with oscillating sliders. I’ll post details on GitHub

Here’s another bug and I think i know the cause because I had the same bug in a program I wrote for the same purpose. The keyer memory playback for register 5 is inoperable. The code for that is KMA, not KM10 because there is only one slot :)

73
Jeff

Sent from my iPhone
________________________________

**Comment by martinbradford** [comment]
OK - keyer bug is fixed, but fixed code is not yet released. If you accept the collaborator invite from GitHub, you'll be able to clone and build for yourself...

Martin
________________________________
